pkg/apis/core/v1beta1/types_shoot.go

```diff
@@ -623,6 +623,9 @@ type KubeletConfig struct {
 	// +optional
 	// Default: 1m
 	ImagePullProgressDeadline *metav1.Duration `json:"imagePullProgressDeadline,omitempty" protobuf:"bytes,12,opt,name=imagePullProgressDeadline"`
+	// FailSwapOn makes the Kubelet fail to start if swap is enabled on the node. (default true).
+	// +optional
+	FailSwapOn *bool `json:"failSwapOn,omitempty" protobuf:"varint,13,opt,name=failSwapOn"`
```

Comment:
Should we default it to `true` in the `defaults.go`?

Reply:
I dont think it is needed, it behaves the same as CPUCFSQuota, which dont have an entry in the defaults.go file